common/laws/00_colonial_affairs.txt

```diff
@@ -177,16 +184,21 @@ law_colonial_exploitation = {
 		colonization
 	}
 	modifier = {
-		building_group_bg_manufacturing_unincorporated_throughput_add = -0.1
-		building_group_bg_plantations_unincorporated_throughput_add = 0.2
-		building_group_bg_rubber_unincorporated_throughput_add = 0.2
-		building_group_bg_logging_unincorporated_throughput_add = 0.2
-		building_group_bg_mining_unincorporated_throughput_add = 0.2
-		state_unincorporated_starting_wages_mult = -0.10
-		building_unincorporated_subsistence_output_mult = -0.10
+		building_group_bg_manufacturing_unincorporated_throughput_add = -0.1	#IMP note; must remove if not patched by Paradox
```

Comment:
you mean the modifier is bugged and applies to all territories or something?

Reply:
Yes, a vanilla bug that we can remove if it comes to it.